src/UXClient/Components/HierarchyNavigation/HierarchyNavigation.ts

```diff
@@ -875,7 +875,7 @@ class HierarchyNavigation extends Component{
         instanceObj.node.classed('tsi-selected', true);
     }
 
-    public drawContextMenu = (contextMenuItems, contextMenuOptions: ContextMenuOptions) => {
+    public drawContextMenu = (contextMenuItems: Array<ContextMenuItems>, contextMenuOptions: ContextMenuOptions) => {
```

Comment:
the "Add" button should be disabled when nothing is clicked (with appropriate style, e.g. lower opacity (.6) cursor default, etc) and enabled when there is any checkbox clicked with appropriate styles